linux-user/main.c

```diff
@@ -71,6 +71,7 @@ char *exec_path;
 char real_exec_path[PATH_MAX];
 
 bool opt_one_insn_per_tb;
+bool swallow_mte_exns;
```

Comment:
Is this initialized anywhere?

Reply:
I didn't explicitly initialize it because I was following the same pattern used for `opt_one_insn_per_tb`. I assume they get initialized unconditionally by the argument-parsing logic.